dll/win32/uxtheme/png.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * PROJECT:     ReactOS uxtheme.dll
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     PNG image support in visual styles
+ * COPYRIGHT:   Copyright 2023 Ethan Rodensky <Splitwirez@gmail.com>
+ */
+
+#include "uxthemep.h"
+
+#ifdef MSSTYLES_ACCEPT_PNG_IMAGES
+#include <ole2.h>
+#include <objidl.h>
+#include <gdiplus.h>
+#include <gdipluscolor.h>
+#include <winreg.h>
+#include <shlwapi.h>
+
+BOOL
+MSSTYLES_TryLoadPng(
```

Comment:
Funnily enough that function is coded in such a way that it can also handle other image formats (those gdiplus can handle), so it may be of interest to find a more generic name here.
(Same comment for the "isPNG" and "pngBitmap" below. Also "pngBitmap" should be renamed "phBitmap" to tell it's a pointer to bitmap handle.)

Reply:
> Between that, and the fact that Vista+ visual styles don't use any image formats other than PNG...I think it would make more sense to keep the function's name as-is.

Ah, this is also what I was wondering: Whether the Vista+ visual styles could also support stuff like JPEGs.

Then just keep the function name as it currently is.